server/dive_tasks/tasks.py

```diff
@@ -430,14 +432,17 @@ def train_pipeline(
 
 
 @app.task(bind=True, acks_late=True, ignore_result=True)
-def convert_video(self: Task, path, folderId, auxiliaryFolderId, itemId):
+def convert_video(self: Task, folderId: str, itemId: str):
     # Delete is true, so the tempfile is deleted when the block closes.
     # We are only using this to get a name, and recreating it below.
     with tempfile.NamedTemporaryFile(suffix=".mp4", delete=True) as temp:
         output_path = temp.name
 
+    input_directory = tempfile.TemporaryDirectory()
```

Comment:
within this file we may in the future want to solidify why we are using `tempfile.TemporaryDirectory()` vs `tempfile.mkdtemp()`  there are very small differences (TemporaryDirectory() will auto remove after context is loss) but I think standardizing it should be a future issue.